### ISSUE TITLE
Check failed: AGInfo::IsNone(*output) Inplace operations (+=, -=, x[:]=, etc) are not supported when recording with autograd.

### ISSUE THREAD

**Issue by Hukongtao** [MashiMaroLjc/YOLO#1]
The error happens in
loss = nd.mean(loss_func(ypre, y))
I don't know the reason
